src/data/h1/tags/actor_variant.yml

```diff
@@ -159,10 +159,21 @@ comments:
     - name: maximum firing distance
       en: The maximum range, in world units, where the unit can fire at a target.
     - name: rate of fire
-      en: >
-        This lets you select a rate of fire within the referenced _weapon's_
-        maximum _rounds per second_. If set to `0`, the actor will fire at the
-        weapon's maximum rate.
+      # Using | here to preserve newlines.
+      en: |
+        - This lets you select a rate of fire limited by the referenced _weapon's_
+          minimum _rounds per second_.
```

Comment:
Semantically, an (un)ordered list should be used when there is a series of steps or options introduced. In this case it's more being used to hold individual facts. I'd like to think of the content within a tag field's table cell being like a mini article section, with multiple paragraphs and lists as needed. I'd like to avoid "bare" lists that aren't introduced with some preceding context, so my suggestions here are either:

* Convert this list of sentences into a multi-paragraph text structure, grouping related points into paragraphs.
* Converting the first point into an introductory paragraph for the following list of points.

An exception to the "bare" lists rule is how we render lists of invader field attributes into these cells when available. Sometimes I've separated those with a horizontal rule `---` if they don't look good with the hand-written cell comments.

Reply:
I've updated the points here and given them and introduction. I initially split them up into points here as there is so many things to think about with this one field that having it all in a paragraph doesn't really work. The separate paragraphs idea isn't off the table, but I'm not sure how effective it would be.